### PR TITLE
Add php 7.0 in the build matrix

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

**Comment by Hywan** [comment]
Ready to merge?

**Comment by jubianchi** [comment]
@Hywan let me add the changelog entry and it will be ok ;)

**Comment by jubianchi** [comment]
ping @Hywan

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # `dev-master`
 
+* [#441](https://github.com/atoum/atoum/pull/441) Add php 7.0 in the build matrix ([@jubianchi][jubianchi])
```
Comment (Hywan) [inline CHANGELOG.md]: s/`php`/`PHP` :-/

**Comment by Hywan** [comment]
You should also rebase on the `master`.

File: CHANGELOG.md (hunk 1 above)
Comment (jubianchi) [inline CHANGELOG.md]: :+1:

**Comment by jubianchi** [comment]
I'll fix the typo, rebase and merge ;)

**Pull request merged**